app/controllers/instance_actors_controller.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
-class InstanceActorsController < ApplicationController
-  include AccountControllerConcern
+class InstanceActorsController < ActivityPub::BaseController
+  vary_by ''
 
-  skip_before_action :check_account_confirmation
-  skip_around_action :set_locale
+  serialization_scope nil
+
+  before_action :set_account
+  skip_before_action :require_functional!
+  skip_before_action :update_user_sign_in
```

Comment:
Is this line still needed? If so, do we need `update_user_sign_in` in any of the ActivityPub controllers?

Reply:
This line is needed, but no, I don't think we need it in any other of the `ActivityPub` controllers so it probably makes sense moving it to `ActivityPub::BaseController`, although I'd say it's out of scope for this PR.